### PR TITLE
Partial fix for the getting started tutorial

### PR DESCRIPTION
**Pull request by fhennig**

### CODE REVIEW

**Comment by fhennig** [comment]
Looks like the guide might still not work entirely (didn't work for me on the first try)

Maybe @Jimvin you can have a look some time?

File: modules/ROOT/pages/getting_started.adoc [hunk 1]
```diff
@@ -259,21 +263,30 @@ The shell should connect automatically to the ZooKeeper server running on the po
...
 === Apache Kafka
-To test Kafka we'll create a topic, and verify that it was created.
-First create the topic with the following command:
+To test Kafka we'll create a topic, and verify that it was created. By default, Stackable enabled TLS on Kafka so we will need to configure the Kafka client with the correct connection properties.
 
 [source,bash]
 ----
 kubectl exec -i -t simple-kafka-broker-brokers-0 -c kafka -- \
-  bin/kafka-topics.sh --bootstrap-server localhost:9092 --create --topic demo
+  bash -c "echo -e \"security.protocol=SSL\nssl.truststore.location=/stackable/tls_internal/truststore.p12\nssl.truststore.type=PKCS12\nssl.truststore.password=changeit\" > /stackable/config.properties"
```
Comment (nightkr) [inline modules/ROOT/pages/getting_started.adoc:271]: Would it make sense to have kafka-op write this file instead?

File: modules/ROOT/pages/getting_started.adoc [hunk 2]
```diff
@@ -259,21 +263,30 @@ The shell should connect automatically to the ZooKeeper server running on the po
 ----
 
 === Apache Kafka
-To test Kafka we'll create a topic, and verify that it was created.
-First create the topic with the following command:
+To test Kafka we'll create a topic, and verify that it was created. By default, Stackable enabled TLS on Kafka so we will need to configure the Kafka client with the correct connection properties.
```
Comment (nightkr) [inline modules/ROOT/pages/getting_started.adoc:266]: This is pretty germlish, but irrelevant if we drop this section entirely.

File: modules/ROOT/pages/getting_started.adoc (hunk 1 above)
Comment (fhennig) [reply modules/ROOT/pages/getting_started.adoc:271]: without nowing much about what's going on, from a docs perspective, yes! :D

**Comment by fhennig** [comment]
Alright, things have been fixed in the meantime

**Pull request closed without merging**